test/features/window.tab.manipulation.feature

```diff
@@ -0,0 +1,22 @@
+@Fast
+@Window
+@Tab
+
+Feature: Window/tab
+  In order to check window/tab manipulation
+  As a developer
+  I want to open/close window/tab and check that it works properly 
+
+  Scenario: Open new window step should open new empty tab
+    Given I open "http://localhost:9000/new-page.html"
+    Then txtNewHeader from main page should be present
+    When I open new window
+    Then div from main page should not be present
+    When I close current tab
```

Comment:
It is semantically more correct to use `And` here (as this is additional action)